evap/staff/tests/test_views.py

```diff
@@ -874,6 +874,24 @@ def test_form(self):
         self.assertIn("uiae", mail.outbox[0].body)
 
 
+class TestSendReviewReminderTemplate(WebTestStaffMode):
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.evaluation = baker.make(Evaluation, course__responsibles=[], state=Evaluation.State.EVALUATED)
+
+    def test_form(self):
+        evaluation_url_tuples = [(self.evaluation, "http://example.com/evaluation/42")]
```

Comment:
the email must link to the textanswer review page (as correctly defined in [send_reminders.py](https://github.com/e-valuation/EvaP/pull/1792/files#diff-1830c52762a53b99f7a49a5006f49587f31202f04708849ad12f8a0fdbfce92b))

Reply:
This was just supposed to be any evaluation with url for testing purposes

Reply:
Ah, I see. From `self.assertEqual(evaluation_url_tuples, ...)` below I assumed that this was the _expected_ list of (evaluation, url) pairs passed to the template.

Could we make it so that you don't pass in the tuples, but instead let the system build them, so that we also test that the tuples are built correctly?

Reply:
should be tested now :D